pkg/watcher/conditionwatcher.go

```diff
@@ -35,12 +35,12 @@ import (
 	azurediskInformers "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/informers/externalversions"
 )
 
-type objectType string
+type ObjectType string
```

Comment:
`ObjectType` doesn't seem to be exposed outside of this package; does it really need to be a package public symbol? Same with its constants below.

Reply:
We need object type when we initialize a new `ConditionWaiter`(e.g. https://github.com/sunpa93/azuredisk-csi-driver/blob/c26a3adf73fba8b2f81129eaa2a79b827495cf8b/pkg/provisioner/crdprovisioner.go#L244)

Reply:
nm. I see that these are exposed outside of the package as a parameter to `NewConditionWaiter`.